requirements.txt

```diff
@@ -1,7 +1,6 @@
 numpy
 protobuf
 tf2onnx
-tensorflow
```

Comment:
cannot find module tensorflow, so we need this?

Reply:
putting this line here will restrict the end user has to use tensorflow, but actually the use could use tensorflow-gpu as well.

Reply:
we have build break, move this to requirements-dev?